02-peer-protocol.md

```diff
@@ -625,7 +625,7 @@ error response if it does not match the onion it sent.
 Otherwise, a receiving node which has an outgoing HTLC canceled by
 `update-fail-malformed-htlc` MUST return an error in the
 `update-fail-htlc` sent to the link which originally sent the HTLC
-using the `failure-code` given and setting the `additional` data to
+using the `failure-code` given and setting the data to
```

Comment:
On second thought, the `paymenthash` is committed to in the HMAC of the `onion`, which we can't check anyway, so nevermind